frontend/apps/monkvision/js/main.mjs

```diff
@@ -44,11 +44,11 @@ async function interceptPageLoadAndPageLoadData() {
         for (const key of Object.keys(dashboardsRaw)) {
             const file = dashboardsRaw[key].split(",")[0], refresh = parseInt(dashboardsRaw[key].split(",")[1].split(":")[1]),
                 name = await i18n.get(`name_${key}`, session.get($$.MONKSHU_CONSTANTS.LANG_ID)), title = await i18n.get(`title_${key}`, session.get($$.MONKSHU_CONSTANTS.LANG_ID));
-            const showDash = securityguard.isAllowed(key)? true : false;
 
-            data.dashboards.push({ name, file, refresh, title, id: key ,showDash});
+            if(securityguard.isAllowed(key))
+                data.dashboards.push({ name, file, refresh, title, id: key });
```

Comment:
New line is not required when there is a single action for a condition! Don't make code more verbose than it needs to be !